packages/githublight/src/index.ts

```diff
@@ -39,23 +35,6 @@ const plugin: JupyterFrontEndPlugin<void> = {
       load: () => manager.loadCSS(style),
       unload: () => Promise.resolve(undefined),
     });
-
-    if (settingRegistry) {
-      settingRegistry
-        .load(plugin.id)
```

Comment:
I'm guessing this is the line that triggered the error in the server log. Is that correct?

Why was this code here to begin with? Was the idea that each theme could have configurable settings in the settings editor?

Reply:
Yes, this was calling the endpoint for settings which as we don't handle raised the schema error we were seeing. This was part of the template from the cookiecutter and I also saw a couple of other extensions (that presumably) use settings so at the beginning I decided to leave it there. 

If in the future we want to have some configurations, we may need to add this code. For now, we can just remove it.

Reply:
That makes sense. Thanks for the context!